manifests/alias.pp

```diff
@@ -7,45 +7,20 @@
 #     }
 #
 define kmod::alias(
-  $source,
-  $ensure = 'present',
-  $file   = '/etc/modprobe.d/aliases.conf',
+  $modulename,
```

Comment:
I'd think `$modulename` would point to the name of the alias, not the source… I'd rather keep it the way it is, at least for now so we can focus on the refactoring itself.